mlflow/tracking/_tracking_service/utils.py

```diff
@@ -171,10 +170,33 @@ def _get_databricks_rest_store(store_uri, **_):
     return RestStore(partial(get_databricks_host_creds, store_uri))
 
 
+def _get_databricks_uc_rest_store(store_uri, **_):
+    from mlflow.exceptions import MlflowException
+    from mlflow.version import VERSION
+
+    global _tracking_store_registry
+    raise MlflowException(
+        f"Detected Unity Catalog tracking URI '{store_uri}'. "
+        f"Setting the tracking URI to a Unity Catalog backend is not supported in the current "
```

Comment:
```suggestion
        "Setting the tracking URI to a Unity Catalog backend is not supported in the current "
```

Can we remove useless f-strings?